reference/release-notes/1.25.0.md

```diff
@@ -0,0 +1,118 @@
+---
+orphan: true
+---
+# 1.25.0
+
+These release notes cover new features and changes in Anbox Cloud 1.25.0.
+
+Anbox Cloud 1.25.0 is a minor release. To understand minor and patch releases, see [Release notes](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/release-notes/release-notes).
+
+```{important}
+The 1.25.0 release contains new versions of the Anbox Cloud snaps and images. There are no charms released for 1.25.0, they are instead planned to be released in 1.25.1.
+```
+
+Please see [Component versions](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/component-versions/) for a list of updated components.
+
+## Requirements
+
+See the [Requirements](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/requirements/) for details on general and deployment specific requirements to run Anbox Cloud.
+
+## New features & improvements
+
+### Anbox Cloud Dashboard
+* The dashboard now includes an onboarding tour for new users. Click *Get Started* on the navigation to take the tour.
+* A new *Images* page is included in the Anbox Cloud dashboard so that you can directly add an image or work with the existing images.<!--AC-3001-->
+* Enhancements to the following forms and pages:
+    - *Create Application*
+    - *Edit Application*
+    - *Create Instance*
+    - *Profile*
+    - *Application details*
+
+### AMS
+* When querying image related information, you can now see the image variant without waiting for the full image to be downloaded.<!--AC-2987-->
+* The GPU type is included in the AMS node API object, to provide better visibility into hardware specifications.<!--AC-3004-->
+
+### Stream Gateway
+* An administrator can revoke a pre-signed ADB connection URL before it expires, to prevent malicious actions. See {ref}`howto-access-instance` for more information.<!--AC-3052-->
+* You can now update the expiration date, description and the timeout values of a shared session using the command:<!--AC-3053-->
+
+        anbox-stream-gateway share update <share_id> --expiry=30m --description=new_description
```

Comment:
If we introduced the command
`anbox-stream-gateway share update <share_id> --expiry=30m --description=new_description`
we may as well introduce  
`anbox-stream-gateway share delete <share_id> `command for more context under the sentence of ``An administrator can revoke a pre-signed ADB connection URL before it expires`, otherwise for clarity and simplicity, let's drop the above command as it has been documented in howto-access-instance page